### PR TITLE
Testing through the interface

### PR DESCRIPTION
**Pull request by base10**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -43,10 +43,11 @@ group :development, :test do
 end
 
 group :test do
-  gem "capybara-webkit", ">= 1.2.0"
+  #gem "capybara-webkit", ">= 1.2.0"
```
Comment (drapergeek) [inline Gemfile]: I don't care in this project just an FYI, we don't ever leave things like this commented in a codebase. Git keeps a history for us.

File: spec/support/features/contacts_helpers.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Features
+  def have_contact(name)
```
Comment (base10) [inline spec/support/features/contacts_helpers.rb:2]: Not a predicate method

File: spec/support/features/contacts_helpers.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    have_css("[data-role='contact']", text: name)
+  end
+
+  def have_message(message)
```
Comment (base10) [inline spec/support/features/contacts_helpers.rb:6]: Not a predicate method

File: spec/features/user_sees_contacts_spec.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "User sees contacts" do
+  scenario "successfully", js: true do
+    contact = create(
```
Comment (drapergeek) [inline spec/features/user_sees_contacts_spec.rb]: Have to agree with hound here.

File: spec/features/user_sees_contacts_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "User sees contacts" do
```
Comment (drapergeek) [inline spec/features/user_sees_contacts_spec.rb]: What benefit does this test provide? You have these same things covered in other specs.

Comment (base10) [inline spec/features/user_sees_contacts_spec.rb]: That was just my jumping-in point. I can kill it now.

**Pull request merged**